core-java-modules/core-java-time-measurements/src/test/java/com/baeldung/time/ElapsedTimeUnitTest.java

```diff
@@ -50,19 +50,27 @@ public void givenRunningTask_whenMeasuringTimeWithStopWatch_thenGetElapsedTime()
         
         assertEquals(true, (2000L <= timeElapsed) && (timeElapsed <= 3000L));
     }
-    
-    @Test
+
+    /*
+     The below test depends on the elapsed time, which isn't ideal in a test.
+     Also, it slows down test execution artificially.
+     */
+    /*@Test
```

Comment:
I think in this case we can rename the entire test to *ManualTest?

Reply:
ok